shell/list/management.cattle.io.user.vue

```diff
@@ -13,14 +14,30 @@ export default {
     ResourceTable,
     Masthead
   },
+  mixins: [ResourceFetch],
+  props:  {
+    loadResources: {
+      type:    Array,
+      default: () => []
+    },
+
+    loadIndeterminate: {
+      type:    Boolean,
+      default: false
+    },
 
+    incrementalLoadingIndicator: {
+      type:    Boolean,
+      default: false
+    },
+  },
   async fetch() {
     const store = this.$store;
     const resource = this.resource;
 
     await store.dispatch(`rancher/findAll`, { type: NORMAN.USER });
 
-    this.allUsers = await store.dispatch(`management/findAll`, { type: resource });
+    this.allUsers = await this.$fetchType(resource);
```

Comment:
We have a big problem with those counts because they don't take that into account right now... The count are directly connected to the raw resource count which is fetched by a getter on the `ResourceLoadingIndicator` component itself.
One way to not show those incorrect numbers is just to go with the `indeterminate` mode, for now, on all of them. 

What do you think?

Reply:
using `indeterminate` where results are filtered so that we don't show wrong counts